Source/Editor/Tools/Terrain/Sculpt/SmoothMode.cs

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) 2012-2023 Wojciech Figat. All rights reserved.
 
 using System;
+using System.Security.Cryptography.X509Certificates;
```

Comment:
Is this include needed?

Reply:
Ah good catch, thanks.  This probably got included from a bad autocomplete along the way.

I discovered there are still some problems with my fix and am working to solve those; I'll remove this in the next commit.